sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -1,286 +1,19 @@
 {
-    "type": "form",
-    "tags": [
-        "common"
-    ],
-    "owner": "andrew.schwenker@gov.bc.ca",
-    "components": [
+    "forms": [
```

Comment:
The forms array must be removed. It is used only by FormsFlow.ai and it will cause an issue in our deployment to TEST.